ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Return error code of mbedtls_aes_crypt_ecb() if failed, when calling it from
```

Comment:
Grammar - "Handle the error if mbedtls_aes_crypt_ecb() fails, when calling it from other modes and ciphers. Fix for #1092 reported by adustm"